sphinx/builders/latex.py

```diff
@@ -292,6 +292,7 @@ def setup(app):
     app.add_config_value('latex_logo', None, None, string_classes)
     app.add_config_value('latex_appendices', [], None)
     app.add_config_value('latex_keep_old_macro_names', True, None)
+    app.add_config_value('latex_use_latex_multicolumn', False, None)
```

Comment:
I thought adding a config setting to influence output by LaTeX writer was simpler than use always ``\sphinxstartmulticolumn`` mark-up but redefine the macro to be standard LaTeX's ``\multicolumn`` if  user has toggled some boolean via ``'sphinxsetup'`` for example.